app/main.py

```diff
@@ -1,2 +1,126 @@
 class Dictionary:
-    pass
+    LOAD_FACTOR = 2 / 3
+    RESIZE = 2
+    DEFAULT_CAPACITY = 8
+
+    def __init__(self):
+        self.capacity = 8
+        self.storage = [[] for _ in range(self.DEFAULT_CAPACITY)]
+        self.length = 0
+
+    def resize(self):
+        saved_storage = self.storage[:]
+        self.length = 0
+        self.capacity *= self.RESIZE
+        self.storage = [[] for _ in range(self.capacity)]
+
+        for item in saved_storage:
+            if len(item) != 0:
+                self.__setitem__(item[0], item[1])
+
+    def __setitem__(self, key, value):
+        if self.length == int(self.capacity * self.LOAD_FACTOR):
+            self.resize()
+
+        hashed_value = hash(key)
+        index = hashed_value % self.capacity
+
+        while True:
+            if not self.storage[index]:
+                self.storage[index] = [key, value, hashed_value]
+                self.length += 1
+                return
+            elif hashed_value == self.storage[index][2] and \
+                    key == self.storage[index][0]:
+                if self.storage[index][2] is True:
```

Comment:
This condition is always False when the condition in line 33 is True.

Reply:
I've fixed